tutorials/lodash/remove-null.md

```diff
@@ -0,0 +1,19 @@
+To remove a `null` from an object with lodash, you can use the `omitBy()` function in combination with the `.isNull` check.
+
+```javascript
+const _ = require('lodash');
+
+const obj = {a: null, b: 'Hello', c: 3, d: undefined};
+
+const result = _.omitBy(obj, _.isNull); // {b: 'Hello', c: 3}
```

Comment:
Let's recommend doing `_.omitBy(obj, v => v === null)`. Because, as you saw, `_.isNull` also catches `undefined`.

Instead, let's add a separate section explaining that, if you want to remove both `null` and `undefined`, you can use `_.isNull` or `_.omitBy(obj, v => v == null)`